scripts/getStatus.js

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License in the project root for license information.
+
+'use strict';
+var execSync = require('child_process').execSync,
+  path = require('path'),
+  fs = require('fs'),
+  glob = require('glob'),
+  _ = require('underscore'),
+  oav = require('openapi-validation-tools');
+
+exports = module.exports;
+exports.globPath = path.join(__dirname, '../', '/**/swagger/*.json');
+exports.swaggers = _(glob.sync(exports.globPath));
+
+var swaggersToProcess = exports.swaggers;
+var finalResult = {};
+var filename = `log_${getTimeStamp()}.log`;
+var logFilepath = path.join(getLogDir(), filename);
+
+function getTimeStamp() {
+  // We pad each value so that sorted directory listings show the files in chronological order
+  function pad(number) {
+    if (number < 10) {
+      return '0' + number;
+    }
+
+    return number;
+  }
+
+  var now = new Date();
+  return pad(now.getFullYear())
+    + pad(now.getMonth() + 1)
+    + pad(now.getDate())
+    + "_"
+    + pad(now.getHours())
+    + pad(now.getMinutes())
+    + pad(now.getSeconds());
+}
+
+function updateResult(spec, errors, updateLog) {
+  if (!finalResult[spec]) {
+    finalResult[spec] = errors;
+  } else {
+    finalResult[spec] = finalResult[spec].concat(errors);
+  }
+  if (updateLog) {
+    writeContent(JSON.stringify(finalResult, null, 2));
+  }
+  return;
+}
+
+function getLogDir() {
+  let logDir = path.join(__dirname, '../', 'output');
+  if (!fs.existsSync(logDir)) {
+    fs.mkdirSync(logDir);
+  }
+  return logDir;
+}
```

Comment:
I am not that familiar with CI, but is there any chance that two instance of scripts run concurrently and try to create the Log folder same time? if yes fs.mkdirSync may throw EEXIST in case of a race condition - right?, if so this can be handled as -

```javascript
  let logDir = path.join(__dirname, '../', 'output');
  try {
    fs.mkdirSync(logDir);
  } catch(e) {
    if ( e.code != 'EEXIST' ) throw e;
  }
  return logDir;
```